octobot_trading/api/exchange.py

```diff
@@ -19,14 +19,15 @@
 
 import octobot_trading.constants
 import octobot_trading.enums
-import octobot_trading.exchanges.connectors.ccxt.enums
+import octobot_trading.exchanges.connectors.ccxt.enums as ccxt_enums
 import octobot_trading.exchanges as exchanges
+import octobot_trading.exchanges.exchange_builder as exchange_builder
```

Comment:
The import philosophy is to import high level module (exchanges, personal_data) when not importing from within the module. The idea here is to prevent changing all imports when we update internal module architecture

Reply:
ah okay, I removed those changes